### ISSUE TITLE
f00f and f011 not connecting to the phone app

### ISSUE THREAD

**Issue by j6kim** [SoftwareDefinedBuildings/PECS-chair#18]
@samkumar99 
I reset the chair & restarted the phone multiple times. The phone app is still not able to connect with the chair. Can Sam look into this issue? It's becoming a major problem for our study as the users start to get annoyed and give up using the chair.

**Comment by gtfierro** [comment]
Have you tried reprogramming the chairs?

**Comment by j6kim** [comment]
no. was there an update to the software recently to fix this problem?

**Comment by gtfierro** [comment]
No, but sometimes this fixes issues not solved by power cycling

**Comment by samkumar** [comment]
Are these chairs consistently more problematic than the other chairs that have shown this issue?

**Comment by j6kim** [comment]
@samkumar99 
This problem is not isolated to f00f and f011. This morning, I inspected 12 chairs. Out of 12, 5 chairs had their phone app disconnected from the chairs. I tried reconnecting with the chair from the app but it didn't work. I had to try a combination of chair reset & phone reboot to fix the problems. Both f00f and f011 are working now. 
Although not ideal, we are going to have a regular check-up on chairs and troubleshoot to prevent users from dropping out. We've got another 7 weeks for this field study. It would be very helpful if you could identify what's causing the failure on chair-phone connection and prevent it.
